lib/screens/test_screen/test_screen.dart

```diff
@@ -6,10 +7,45 @@ class TestScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: SingleChildScrollView(
-        child: Center(
-          child: CommonText(
-            text: 'my_profile'.tr,
+      body: SafeArea(
+        child: SingleChildScrollView(
+          child: Center(
+            child: Column(
+              children: <Widget>[
+                SizedBox(height: DEVICE_HEIGHT * 0.1),
+                CommonButton(
+                  buttonStyle: DefaultButtonStyles().defaultSmallButton(),
+                  text: 'Login'.tr,
```

Comment:
The key you've added in en and ar files is `login` not `Login`
Let's fix it here